### DISCUSSION TITLE
What is the `rusqlite-le` crate

### DISCUSSION THREAD

**Discussion by dnaka91** [rusqlite/rusqlite#1220] [Questions] [answered]
I saw that there is a `rusqlite-le` crate available that links back to this repository and seems to be used by several other crates as dependency.

But I couldn't find any mention of that crate anywhere in this project, so my best guess is that the `le` part means something like **Legacy Edition**?

Would just be curious what the crate is about, whether and when it should be used (or avoided). Especially as there are some interesting crates that depend on it, whereas I myself depend on the main `rusqlite` crate instead.

**Comment by thomcc** [comment, marked as answer]
It's unrelated to this project and not maintained by anybody I recognize. It's presumably some fork someone decided they needed.

I wouldn't recommend anybody use it.

**Reply by dnaka91** [reply]
Thank you for the quick reply 🚀

After a bit of digging, it seems to be published by the same person that owns the only crate using it. Said user is even banned in several places, so I better stay away from that whole package (even though the project is cool).

**Comment by gwenn** [comment]
Should be related to #868 => `le` means loadable extension...
